Update Skeleton Instructions for Development Guide Project Setup (#97)
cisagov/development-guide: project_setup/skeleton-list.md

to make starting new repositories quicker and to simplify maintaining
configurations and dependencies across the organization.

> The [`gh-skeleton`](https://github.com/cisagov/gh-skeleton) is

Reviewer: This doesn't make sense as it is written. Also please use a reference-style link and convert all uses of "gh-skeleton" in line with my previous suggestion.

```suggestion
The [`gh-skeleton`](https://github.com/cisagov/gh-skeleton) project is
```

> a custom tool that provides the ability to easily start new projects

Reviewer 2: ```suggestion
a GitHub CLI (`gh`) extension that provides the ability to easily start new projects
```

> from existing library of skeleton repositories.

Reviewer: ```suggestion
from an existing library of skeleton repositories.
```

- To skeletonize an existing repository, please see the guide to
[skeletonize an existing repository](skeletonize-existing-repository.md).
- To skeletonize a new repository, please see the guide to

> [start a new repository from a skeleton](README.md).

Reviewer: Are we able to use a link fragment here to link to the appropriate section?

> [start a new repository from a skeleton](https://github.com/cisagov/gh-skeleton).

Reviewer 2: This is a better link:
```suggestion
[start a new repository from a skeleton](README.md).
```

## List available skeletons with the command-line tool ##

> After you've installed the [`gh` CLI](https://github.com/cli/cli) followed

Reviewer: If we're going to link out I would prefer we use a reference-style link and update all uses of "gh" to use it where appropriate.

```suggestion
After you've installed [`gh`](https://github.com/cli/cli) followed
```

by the [`gh-skeleton`](https://github.com/cisagov/gh-skeleton) extension,
the tool will be available to you via the command line. To see a list of
available skeletons, use the following command:

> ```bash

Reviewer: ```suggestion
```console
```

> gh skeleton list

Reviewer: ```suggestion
$ gh skeleton list
```
